src/Select.jsx

```diff
@@ -1082,7 +1082,7 @@ export default class Select extends React.Component {
       let selectedValueNodes = [];
       let limitedCountValue = value;
       let maxTagPlaceholderEl;
-      if (maxTagCount && value.length > maxTagCount) {
+      if (Number.isInteger(maxTagCount) && value.length > maxTagCount) {
```

Comment:
IE does not support `Number.isInteger`,  I think check `maxTagCount  !== undefined` is enough.

Reply:
well

Reply:
Don't need close, just change `Number.isInteger(maxTagCount)` to `maxTagCount !== undefined`

Reply:
I thought I should create a new one cause I rewrite the last commit...